### PR TITLE
Add browser network audits

### PR DESCRIPTION
**Pull request by FelixZilber**
Adding network logs audits and log it to report

### CODE REVIEW

File: src/commons/Reporter.ts [hunk 1]
```diff
@@ -45,18 +45,72 @@ export namespace Reporter {
...
+      browser.on('Network.responseReceived', (params: any) => {
+        if (params.type.toLowerCase() === 'xhr' || params.type.toLowerCase() === 'fetch') {
+          networkActivity.push({
+            url: params.response.url,
+            status: params.response.status,
+            headers: params.response.headers,
+          });
+        }
+      });
+    }
```
Comment (AlexeyCL) [inline src/commons/Reporter.ts]: suggest renaming to StartNetworkAudit

File: src/commons/Reporter.ts [hunk 2]
```diff
@@ -45,18 +45,72 @@ export namespace Reporter {
...
+   * In beforeTest hook:
+   *      Reporter.enableNetworkAudits()
+   * In afterTest hook:
+   *      Reporter.addAttachment('Network Logs', { https: networkActivity }, 'application/json');
+   *    already integrated in Reporter.closeStep method in case of test failure
+   */
+  export function enableNetworkAudits(): void {
+    if (browser.capabilities.browserName === 'chrome') {
+      // @ts-ignore
+      browser.cdp('Network', 'enable');
```
Comment (AlexeyCL) [inline src/commons/Reporter.ts]: pls add jdoc, what kind of audit we taken

File: src/commons/Reporter.ts [hunk 3]
```diff
@@ -45,18 +45,72 @@ export namespace Reporter {
...
+   *
+   * Example of usage:
+   * In beforeTest hook:
+   *      Reporter.enableNetworkAudits()
+   * In afterTest hook:
+   *      Reporter.addAttachment('Network Logs', { https: networkActivity }, 'application/json');
+   *    already integrated in Reporter.closeStep method in case of test failure
+   */
+  export function enableNetworkAudits(): void {
+    if (browser.capabilities.browserName === 'chrome') {
```
Comment (AlexeyCL) [inline src/commons/Reporter.ts]: add  >    networkActivity.splice(0, networkActivity.length);
before enabling

File: src/commons/Reporter.ts (hunk 1 above)
Comment (FelixZilber) [reply src/commons/Reporter.ts]: changed to startNetworkAudit

File: src/commons/Reporter.ts (hunk 2 above)
Comment (FelixZilber) [reply src/commons/Reporter.ts]: added "   * Will log xhr and fetch responses
"

**Pull request merged**